### DISCUSSION TITLE
Be able to import external examples (JSON files)

### DISCUSSION THREAD

**Discussion by oliviernguyenquoc** [ApicurioArchive/apicurio-studio#2468] [Ideas]
By using the tool, I feel like the need to have a way to import external JSON example like that:

```
openapi: 3.1
info:
  title: API Pastry - 2.0
  version: 2.0.0
  description: API definition of API Pastry sample app
  license:
    name: MIT License
    url: https://opensource.org/licenses/MIT
components:
  examples:
    pastries_json:
      summary: Some delicious pastries
      $ref: './examples/an_external.json'
   single_pastry:
     summary: A unique delicious pastry
     $ref: './examples/an_external.json#/0'
```

In fact, in the development process, It would be great to have example in seperate JSON file to be able to both import them in the OpenAPI specs and to be able to use them in automatic tests 

I don't know the perfect behaviour (one shot import or create a link that's updating the spec based on JSON changes) but there's somehting to dig into, IMO.

**Comment by EricWittmann** [comment]
Thanks for the feedback.  I like this perspective - more of an example-driven development approach.
